include/tensor.cuh

```diff
@@ -398,7 +398,7 @@ public:
      * @param b provided tensor
      * @return least squares solution (overwrites b)
      */
-    void leastSquares(DTensor &b);
+    void leastSquaresBatched(DTensor &b);
```

Comment:
If I get this right, $A\in\mathbb{R}^{m\times n\times n}$ is a matrix ($m\geq n$), $b\in\mathbb{R}^{n\times 1\times k}$ is a bunch of vectors $b = (b_1, b_2, \ldots, b_k)$ (packed in a tensor), and this function performs the operation $A^{\dagger}b = (A^{\dagger}b_1, \ldots, A^{\dagger}b_k)$. Is this right? We should make the documentation a bit more clear in any case.